src/main/java/tars/Response.java

```diff
@@ -0,0 +1,120 @@
+package tars;
+
+import tars.commands.AddCommand;
+import tars.commands.DeleteCommand;
+import tars.commands.FindCommand;
+import tars.commands.MarkCommand;
+
+import tars.exceptions.TarsException;
+
+import tars.tasks.TaskList;
+
+/**
+ * Handles the generation and formatting of responses for user interactions.
+ *
+ * <p>The {@code Response} class provides methods to display introductory and outro messages,
+ * format responses based on user commands, and handle exceptions gracefully. It leverages
+ * various command classes ({@link MarkCommand}, {@link DeleteCommand}, {@link AddCommand})
+ * to execute user commands and generate appropriate responses.</p>
+ */
+public class Response {
```

Comment:
Maybe it would be more managable to separate the UI and the parser into separate classes?